modules/openy_traction_rec_import/src/TractionRecFetcher.php

```diff
@@ -203,14 +203,21 @@ public function fetchProgramAndCategories():void {
     $programs = [];
     $categories = [];
     foreach ($result['records'] as $key => $category_tag) {
-      // It's confusing, but in Open Y terms we we have vice verse structure:
+      // It's confusing, but in Open Y terms we have a reverse structure:
```

Comment:
Well, instead of asking for apologies, can we extend a bit docs and make it less confusing?

Reply:
Will go in docs with the data structure.